### ISSUE TITLE
[pre-release] KeeShare options don't show certificate panel

### ISSUE THREAD

**Issue by franzlst** [keepassxreboot/keepassxc#2761]
I'm using the pre-release 2.4.0-beta2. In the settings for KeeShare, there is no panel for the "Own certificate", as it is mentioned and shown in the documentation of the feature branch: https://github.com/hicknhack-software/keepassxc/blob/feature/sharing_groups/docs/QUICKSTART.md#enable-sharing

This is how the settings look for me:
![keeshare](https://user-images.githubusercontent.com/1144966/53683329-e3577f00-3cff-11e9-9779-1ee4485ced54.PNG)

I'm not sure, whether this was dropped somewhere in the development. However, when I want to synchronize a group, I get the follwing warning: "We cannot verify the source of the shared container because it is not signed. [...]". So I expect there is the pssibility to sign the container.

## Debug Info
KeePassXC - Version 2.4.0-beta2
Build Typ: PreRelease
Revision: 9bc20f0

Bibliotheken:
- Qt 5.12.1
- libgcrypt 1.8.4

Betriebssystem: Windows 10 (10.0)
CPU-Architektur: x86_64
Kernel: winnt 10.0.17763

Aktivierte Erweiterungen:
- Auto-Type
- Browser-Integration
- SSH-Agent
- KeeShare (only unsigned sharing)
- YubiKey

**Comment by droidmonkey** [comment]
I forgot to build in the protected share for beta 2. It'll be in the final

**Comment by librejoker** [comment]
Own certificate panel is still missing for me in Arch and AppImage version. Also when selecting an certified keeshare-file (kdbx.share) I get an error stating it isn't supported.

**Comment by droidmonkey** [comment]
KeeShare secure is not supported in AppImage at this time.

**Comment by benjaminfrank** [comment]
I have the same issue under OSX using the official 2.4.0 release: the panel to generate a certificate is missing. Should I open another issue for that?

**Comment by phoerious** [comment]
No, that's because homebrew does not provide quazip5. I guess we may have to submit that ourselves.

**Comment by jkolo** [comment]
In arch I also haven't certificate panel. This is arch problem or I just have to install some library?

**Comment by phoerious** [comment]
You need to compile with -DWITH_XC_KEESHARE_SECURE=ON. It's not included in -DWITH_XC_ALL as of yet.

**Comment by toussa** [comment]
I have the same issue with KeePassXC 2.6.2

I got the package from debian bullseye repository directly.

I normally have access to keeshare in that version.

Here is the debug info from `about` section (in french) : 

```
KeePassXC - Version 2.6.2
Révision : e9b9582

Qt 5.15.2
Le mode débogage est désactivé.

Système d’exploitation : Debian GNU/Linux bullseye/sid
Architecture de l’unité centrale : x86_64
Noyau : linux 5.10.0-2-amd64

Extensions activées :
- Saisie automatique
- Intégration aux navigateurs
- Agent SSH
- KeeShare (partage non signé uniquement)
- YubiKey
- Intégration au Secret Service

Bibliothèques cryptographiques :
 libgcrypt 1.8.7

```

**Comment by droidmonkey** [comment]
I don't speak French, but Google told me you don't have signed shares capability..........

**Comment by toussa** [comment]
Yes, indeed, it says so (`- KeeShare (partage non signé uniquement)`).
Let me rephrase it : Why don't we have signed share capabilities in debian ? Is it because some external libraries are missing or is it just not supported on debian bullseye (last version) ?
And why do we have access to the "import certificate" section if it's unsigned keeshare ?
Thanks

**Comment by phoerious** [comment]
Ask Debian
You need Quazip 0.9 as a dependency for it to work.

**Comment by toussa** [comment]
Thanks :)
In Debian we have [libquazip5-dev (0.9.1-1)](https://packages.debian.org/fr/bullseye/libquazip5-dev) and [libquazip5-1 (0.9.1-1)](https://packages.debian.org/fr/bullseye/libquazip5-1). However installing it does not solve the issue and we're still "unsigned keeshare only".

**Comment by phoerious** [comment]
It needs to be built with it. There is nothing we can do. It's up to the Debian maintainers.

**Comment by toussa** [comment]
Ok, I get it now. Thanks a lot for your help !
